src/checksum/checksum.csproj

```diff
@@ -32,6 +34,7 @@
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>

```

Comment:
Same as the previous comment, this property should probably not be here in this conditional group.

Reply:
I will get this fixed up.